packages/is-shallow-equal/index.js

```diff
@@ -0,0 +1,90 @@
+var isArray = Array.isArray,
+	keys = Object.keys;
+
+/**
+ * Returns true if the two objects are shallow equal, or false otherwise.
+ *
+ * @param {Object} a First object to compare.
+ * @param {Object} b Second object to compare.
+ *
+ * @return {Boolean} Whether the two objects are shallow equal.
+ */
+function isShallowEqualObjects( a, b ) {
+	if ( a === b ) {
+		return true;
+	}
+
+	var aKeys = keys( a ),
+		bKeys = keys( b ),
+		length = aKeys.length;
```

Comment:
Can't we use separate `var` for each one?

Reply:
>Can't we use separate `var` for each one?

We can, but my obsessive-yet-likely-often-misled interpretations of benchmark results had me do very specific code style 😄

Reply:
But in reality, these are not that important because in general these get mangled, minified in the app bundles.

Reply:
Fair point, at least for web bundles. There's still application in a Node script, which would use this code as written.

I should note that I'm not even certain this was the reason for the particular style in this case, just a general note that common style best practices which are encouraged more for maintainability's sake were deemphasized here. The file would probably need a fair few number of `eslint-disable`.